README.md

```diff
@@ -1,64 +1,92 @@
 
+### Meeting notes
 
-## 
-**Objective**
+Meeting notes are [maintained in a Google Doc](https://docs.google.com/document/d/1KQalBRzfRBvsqh73JUYfp1KG-AJdXcv2Z8LTIFoQP8c/edit#). If attending please add your name, and if a returning attendee, please change the color of your name from gray to black.
 
-[What is to be achieved with this initiative]
+### Areas that need contributions
 
-[OKRs - OPTIONAL]
+- WG Documentation
+- Others TBD
 
+### Where to file issues
 
-## 
-**Scope**
+Issues can be reviewed and filed [here](https://github.com/ossf/wg-endusers/issues)
 
-[What is in and out of scope]
+## Objective
 
+The End User Working Group aims to ensure that the distinct and impactful voice of end users is heard in the development and delivery of the technical vision of OpenSSF.
 
-## 
-**Prior Work**
+## Vision
 
+The End Users Working Group (WG) represents the interests of public and private sector organizations that primarily consume open source rather than produce it.
 
+We strive to:
 
-*   List of prior and/or related projects
+- Ensure that the use cases for end user consumption of open- source software are understood and factored into OSSF programs.
+- Provide the resources required by End Users to develop and implement more efficient and effective strategies, processes, tools, best practices and solutions that secure software supply-chains
+- Provide a forum for learning from the experience and insights of peers
+- Create an End Users working group, with representation from key private industry, public sectors, and multiple geographical regions.
+- Establish user representation and active participation in OpenSSF working groups and leadership, both in the TAC and the Governing Board
 
-# 
-**Get Involved**
+## Scope
 
-*   Official communications occur on the [ADD LINK TO YOUR WG MAILING LIST] (ex: https://lists.openssf.org/g/openssf-tac/topics).  \
-[Manage your subscriptions to Open SSF mailing lists](https://lists.openssf.org/g/main/subgroups).
-*   [Add Slack information if availabable]
+Based on the objective, mission, and goals above, we look to deliver specific delivereables:
 
-### 
-Quick Start
+- _Ensure that the use cases for end users consumption of open- source software is understood and factored into OSSF programmes._
+  - Refine Personas document
+  - Develop a high level architecture and threat model of an end user within the supply chain
+  - Identify controls/checks of interest to consumers when ingesting OSS / vendor software (OSSF Scorecard)
+- _Provide the resources required by End Users to develop and implement more efficient and effective strategies, processes, tools, best practices and solutions that secure software supply-chains_
+  - Bring more end users and their perspectives into the End Users working group, ensuring representation from as many industries, sectors and geographical regions as possible
```

Comment:
End User/end user appear in the same para.

Reply:
I think line 49 is the correct usage though updated Working Group to match the capitalization used elsewhere.